packages/core/src/client/index.ts

```diff
@@ -1,31 +1,113 @@
 import type { Schema } from "@/internal/types.js";
 import type { ReadonlyDrizzle } from "@/types/db.js";
+import { promiseWithResolvers } from "@ponder/common";
+import type { QueryWithTypings } from "drizzle-orm";
+import { type PgSession, pgTable } from "drizzle-orm/pg-core";
 import { createMiddleware } from "hono/factory";
+import { streamSSE } from "hono/streaming";
 
+const status = pgTable("_ponder_status", (t) => ({
+  chainId: t.bigint({ mode: "number" }).primaryKey(),
+  blockNumber: t.bigint({ mode: "number" }),
+  blockTimestamp: t.bigint({ mode: "number" }),
+  ready: t.boolean().notNull(),
+}));
+
+/**
+ * Middleware for `@ponder/client`.
+ *
+ * @param db - Drizzle database instance
+ *
+ * @example
+ * ```ts
+ * import { db } from "ponder:api";
+ * import { Hono } from "hono";
+ * import { client } from "ponder";
+ *
+ * const app = new Hono();
+ *
+ * app.use(client({ db }));
+ *
+ * export default app;
+ * ```
+ */
 export const client = ({ db }: { db: ReadonlyDrizzle<Schema> }) => {
+  // @ts-ignore
+  const session: PgSession = db._.session;
```

Comment:
Does this mean all queries will use the same connection? Seems important to use a pool here.

Reply:
Variable wasn't being used, deleted